cclib/parser/nwchemparser.py

```diff
@@ -1261,6 +1282,27 @@ def extract(self, inputfile, line):
                 self.append_attribute("vibirs", float(line.split()[5]))
                 line = next(inputfile)  # next line
             self.set_attribute("vibfreqs", vibfreqs)
+        # properties related to symmetry
+        if line.strip().find('symmetry #') != -1:
+            symmno = int(line.strip().split()[-1][0:-1])
+            self.set_attribute('symmno', symmno)
+
+        # Grab point group
+        if line.strip().find('symmetry detected') != -1:
+            point_group = line.strip().split()[0]
+            self.set_attribute('point_group', point_group)
+
+        # Grab rotational constants (convert cm-1 to GHz)
+        if line.strip().startswith('A='):
+            roconst, rotemp = [], []
+            roconst.append(float(line.strip().split()[1])*29.9792458)

```

Comment:
This should use `utils.converter` and `line.split()` should be stored in a variable to avoid multiple `split` calls.